### PR TITLE
Fixes #22: Use andromatic for `emulator` command

### LINKED ISSUE #22: Use andromatic for `emulator` command
We expect the user to have an `emulator` command in the `PATH` here:

https://github.com/tweaselORG/cyanoacrylate/blob/886282d5550512f8a0a6c2529fcc9610642c5845/src/index.ts#L473

This currently isn't even documented. Instead, we should automatically install that using andromatic.

### PR DESCRIPTION
**Pull request by baltpeter**

### CODE REVIEW

File: src/util.ts [hunk 1]
```diff
@@ -150,3 +152,12 @@ export const fileExists = async (path: string) => {
...
     }
 };
+
+// We can't use `runAndroidDevTool` for this because that can only give you the process if you `await` it, which we
+// don't want.
+export const startEmulator = async (name: string, args: string[]) => {
+    const { env } = await ensureSdkmanager();
+    const toolPath = await getAndroidDevToolPath('emulator');
+
+    return () => execa(toolPath, ['-avd', name, ...args], { env, stdio: 'pipe' });
```
Comment (zner0L) [inline src/util.ts:162]: I am not sure why you set this to `pipe`, but I think the output would be useful:

```suggestion
    return () => execa(toolPath, ['-avd', name, ...args], { env, stdio: 'inherit' });
```

**Review by zner0L — CHANGES_REQUESTED** [review]
Starting the emulator fails for me:
```
INFO    | Android emulator version 32.1.13.0 (build_id 10086546) (CL:N/A)
INFO    | AVD tweasel-test has path /Users/zner0L/.android/avd/tweasel-test.avd
INFO    | trying to check whether /Users/zner0L/Library/Caches/andromatic is a valid sdk root
WARNING | /Users/zner0L/Library/Caches/andromatic/system-images/android-33/google_apis/x86_64/ is not a valid directory.
WARNING | emulator has searched the above paths but found no valid sdk root directory.
PANIC: Cannot find AVD system path. Please define ANDROID_SDK_ROOT
```

**Comment by baltpeter** [comment]
The error is https://github.com/tweaselORG/andromatic/issues/7 (or you cleared your cache directory since creating the emulator). We won't be fixing that today.

File: src/util.ts (hunk 1 above)
Comment (baltpeter) [reply src/util.ts:162]: Oops, that is left over from debugging.

Comment (zner0L) [reply src/util.ts]: The default is `pipe`, so removing it does not show the output either.

Comment (baltpeter) [reply src/util.ts]: I don't want to show the output…

Comment (baltpeter) [reply src/util.ts]: But that does explain why I didn't see any output during debugging. :D

**Review by zner0L — APPROVED** [review]
It works with emulators created with `andromatic`. Great!

**Pull request merged**